### DISCUSSION TITLE
Recommended Version?

### DISCUSSION THREAD

**Discussion by justinphelps** [familytree365/genealogy#1667] [Q&A] [answered]
It seems that  v3.0.2-alpha was the last version to pass tests. Is that the recommended / stable version? 
If not, which version should I be using?

**Comment by curtisdelicata** [comment]
v4 works fine. the tests need to be changed, we will sort out all the tests and add new ones soon.

**Comment by curtisdelicata** [comment]
I can announce that tests now all pass fine after some modifications and small updates.

**Comment by curtisdelicata** [comment]
Please see familytree365/backend and familytree365/frontend we abandoned the Laravel Enso version.

**Comment by curtisdelicata** [comment]
We are now back to working with Enso. We plan to adapt Nuxt and all our changes and improvements done to the alternative created.

**Comment by curtisdelicata** [comment]
Version 7 is making progress, the backend tests now complete for core features (we have not written tests for unique parts yet).

Nuxt conversion expected completion is next Monday.

**Comment by curtisdelicata** [comment]
Version 8 within a few days.

**Comment by curtisdelicata** [comment]
Please use version 10.

**Comment by curtisdelicata** [comment, marked as answer]
Please use version 11 with Laravel 10 support. Tests pass.
